### DISCUSSION TITLE
Append react-joyride__overlay to different container

### DISCUSSION THREAD

**Discussion by maxcao13** [gilbarbara/react-joyride#897] [Q&A] [answered]
Hi is it possible to allow a prop to append the overlay to your own portal? 

It looks weird like this:
![image](https://user-images.githubusercontent.com/69827283/227055915-bddc3459-382c-43ce-b4ba-193d981f2fc4.png)

**Comment by gilbarbara** [comment, marked as answer]
Hey @maxcao13 
That's not something I'm planning to add right now.
